shape-calculator/src/components/ShapeDisplay.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import '../styles/shapes.css'
+
+const ShapeDisplay = (props) => {
+    const shapeStyle = {
```

Comment:
Descrtucting props here might be useful so you don't have to type props over and over